### PR TITLE
TASK: Refactor absolute namespaces in Media / Tests

### PR DESCRIPTION
**Pull request by johannessteu**

### CODE REVIEW

File: TYPO3.Media/Tests/Functional/AbstractTest.php [hunk 1]
```diff
@@ -10,11 +10,16 @@
  * information, please view the LICENSE file which was distributed with this
  * source code.
  */
+use TYPO3\Flow\Persistence\PersistenceManagerInterface;
+use TYPO3\Flow\Resource\Resource;
```
Comment (kitsunet) [inline TYPO3.Media/Tests/Functional/AbstractTest.php]: Should be PersistentResource

File: TYPO3.Media/Tests/Functional/AbstractTest.php [hunk 2]
```diff
@@ -61,11 +66,11 @@ protected function getMockResourceByImagePath($imagePathAndFilename)
      * file_put_content('resource://' . $hash) before
      *
      * @param string $hash
-     * @return \TYPO3\Flow\Resource\Resource
+     * @return Resource
      */
     protected function createMockResourceAndPointerFromHash($hash)
     {
-        $mockResource = $this->getMockBuilder('TYPO3\Flow\Resource\Resource')->setMethods(array('getHash', 'getUri'))->getMock();
+        $mockResource = $this->getMockBuilder(Resource::class)->setMethods(array('getHash', 'getUri'))->getMock();
```
Comment (kitsunet) [inline TYPO3.Media/Tests/Functional/AbstractTest.php]: Should be PersistentResource

File: TYPO3.Media/Tests/Functional/AbstractTest.php [hunk 3]
```diff
@@ -61,11 +66,11 @@ protected function getMockResourceByImagePath($imagePathAndFilename)
      * file_put_content('resource://' . $hash) before
      *
      * @param string $hash
-     * @return \TYPO3\Flow\Resource\Resource
+     * @return Resource
```
Comment (kitsunet) [inline TYPO3.Media/Tests/Functional/AbstractTest.php]: Should be PersistentResource

File: TYPO3.Media/Tests/Unit/TypeConverter/ImageInterfaceConverterTest.php [hunk 4]
```diff
@@ -10,14 +10,24 @@
...
+use TYPO3\Flow\Object\ObjectManagerInterface;
+use TYPO3\Flow\Persistence\PersistenceManagerInterface;
+use TYPO3\Flow\Property\PropertyMappingConfiguration;
+use TYPO3\Flow\Property\TypeConverter\PersistentObjectConverter;
+use TYPO3\Flow\Reflection\ReflectionService;
+use TYPO3\Flow\Tests\UnitTestCase;
+use TYPO3\Media\Domain\Model\Image;
+use TYPO3\Media\Domain\Model\ImageInterface;
+use TYPO3\Media\TypeConverter\ImageInterfaceConverter;
+use TYPO3\Flow\Resource\Resource;
```
Comment (kitsunet) [inline TYPO3.Media/Tests/Unit/TypeConverter/ImageInterfaceConverterTest.php]: Should be PersistentResource

File: TYPO3.Media/Tests/Unit/Domain/Model/VideoTest.php [hunk 5]
```diff
@@ -24,7 +25,7 @@ class VideoTest extends UnitTestCase
      */
     public function dimensionsDefaultToMinusOneOnConstruct()
     {
-        $mockResource = $this->createMock('TYPO3\Flow\Resource\Resource');
+        $mockResource = $this->createMock(Resource::class);
```
Comment (kitsunet) [inline TYPO3.Media/Tests/Unit/Domain/Model/VideoTest.php]: Should be PersistentResource

File: TYPO3.Media/Tests/Unit/Domain/Model/VideoTest.php [hunk 6]
```diff
@@ -11,6 +11,7 @@
  * source code.
  */
 
+use TYPO3\Flow\Resource\Resource;
```
Comment (kitsunet) [inline TYPO3.Media/Tests/Unit/Domain/Model/VideoTest.php]: Should be PersistentResource

**Comment by johannessteu** [comment]
Fixed

**Comment by kitsunet** [comment]
This still need PersistentResource in a couple of places. Will fix later if you don't.

**Comment by kitsunet** [comment]
That should be all. Will merge when Travis is done.

**Pull request merged**